be/src/column/json_column.cpp

```diff
@@ -260,6 +262,41 @@ void JsonColumn::append_default() {
     }
 }
 
+void JsonColumn::append_selective(const Column& src, const uint32_t* indexes, uint32_t from, uint32_t size) {
+    const auto* other_json = down_cast<const JsonColumn*>(&src);
+    if (other_json->is_flat_json() && !is_flat_json()) {
+        // only hit in AggregateIterator (Aggregate mode in storage)
+        DCHECK_EQ(0, this->size());
+        std::vector<ColumnPtr> copy;
+        for (const auto& col : other_json->_flat_columns) {
+            copy.emplace_back(col->clone_empty());
+        }
+        set_flat_columns(other_json->flat_column_paths(), other_json->flat_column_types(), copy);
+    }
+
+    if (is_flat_json()) {
+        DCHECK(src.is_object());
+        DCHECK_EQ(_flat_column_paths.size(), other_json->_flat_column_paths.size());
+        DCHECK_EQ(_flat_column_paths.size(), other_json->_flat_column_types.size());
+        DCHECK_EQ(_flat_column_types.size(), other_json->_flat_column_paths.size());
+
+        DCHECK_EQ(_flat_columns.size(), other_json->_flat_columns.size());
+
+        for (size_t i = 0; i < _flat_column_paths.size(); i++) {
+            DCHECK_EQ(_flat_column_paths[i], other_json->_flat_column_paths[i]);
```

Comment:
if column1 from load A, all data are int types . column2 from load B, all data are string type. 
what's the data type in flat_column_path?

Reply:
will be `TYPE_JSON`